openreview/conference/matching.py

```diff
@@ -769,9 +769,9 @@ def _build_config_invitation(self, scores_specification):
                 'signatures': {'values': [self.conference.get_id()]},
                 'content': {
                     'title': {
-                        'value-regex': '.{1,250}',
+                        'value-regex': '(?!^ +$)^[a-zA-Z0-9-_ ]{1,250}$',
```

Comment:
could you try to add a test, probably in test_matching.py, create an Assignment Configuration note after calling `conference.setup_committee_matching`.

Reply:
@purujitgoyal could you add a test here?